### PR TITLE
fixed sizing of tables

### PR DESCRIPTION
**Pull request by liron58**
Added the font size feature to work on tables too. 
#130 
image
image
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6483 +1,8 @@
 {
```
Comment (norbit8) [inline package-lock.json]: This file is in `.gitignore`, you should not touch it, unless you have a good reason (?)

File: content_scripts/storageLoader.js [hunk 2]
```diff
@@ -69,18 +69,36 @@ const makeCursorBigger = () => {
 }
 
 const setFontSize = (sizeValue) => {
+    const dashboard = window.moodleBoosterDashboard;
     switch (sizeValue) {
         case "1":
             document.getElementsByTagName("body")[0].style.fontSize = "30px";
+            setTableFontSize('larger');
```
Comment (norbit8) [inline content_scripts/storageLoader.js]: larger is smaller than large?

**Review by norbit8 — CHANGES_REQUESTED** [review]
Nice job man, please see the comments 🥇

File: content_scripts/oneClickDownload.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+
```
Comment (norbit8) [inline content_scripts/oneClickDownload.js]: I think you've mixed this file from another PR of yours, right?

**Review by norbit8 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (liron58) [reply package-lock.json]: Done.

File: content_scripts/storageLoader.js (hunk 2 above)
Comment (liron58) [reply content_scripts/storageLoader.js]: Youre right fixed it

**Comment by idan2468** [comment]
Until Liron decides to fix it I'm closing the PR

**Pull request closed without merging**